### DISCUSSION TITLE
Remix example uses renderToString

### DISCUSSION THREAD

**Discussion by jamesarosen** [tw-in-js/twind#408] [Q&A]
The [Remix example app](https://github.com/tw-in-js/twind/tree/main/examples/with-remix) [uses `react-dom/server`'s `renderToString`](https://github.com/tw-in-js/twind/blob/deb7362ad4093fb547531b617855728975d61a2d/examples/with-remix/app/entry.server.tsx#L13), which causes Remix to render the whole page to a string server-side.

The [example in remix-run/examples](https://github.com/remix-run/examples/tree/main/twind) [uses `renderToPipeableStream`](https://github.com/remix-run/examples/blob/fb7fee55fd37f1595b1513e6d24279a070c273bc/twind/app/entry.server.tsx#L42-L43).

As far as I can tell, `inline` itself consumes the input buffer (or string) so it can extract styles and write the `<style>` tag.

Is one approach preferred over the other? Why?

Possibly related: https://github.com/tw-in-js/twind/discussions/147

**Comment by sastan** [comment]
Thank you for the heads-up. I will adjust the remix example.
